torch/_dynamo/variables/dicts.py

```diff
@@ -613,15 +613,11 @@ def is_matching_cls(cls):
         if (
             issubclass(cls, collections.OrderedDict)
             and cls.__init__ is collections.OrderedDict.__init__
+            and cls.__getitem__ is collections.OrderedDict.__getitem__
+            and cls.__setitem__ is collections.OrderedDict.__setitem__
```

Comment:
I don't think this is true. ```CustomizedDictVariable``` is used to capture a python class extends from ```dict``` or ```OrderedDict```, so overwriting ```__getitem__``` and ```__setitem__``` is allowed.

Reply:
Yeah you are right - this and the `call_function` change both seem wrong, inlining seems like the right approach; I was still having trouble following what's going on but I rewatched all the deep dive videos and I think it makes some more sense now.